presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -380,8 +381,11 @@ public static LocalDateTime fromPrestoTimestamp(long value)
         return Instant.ofEpochMilli(value).atZone(UTC).toLocalDateTime();
     }
 
-    public static Optional<ColumnMapping> jdbcTypeToPrestoType(ConnectorSession session, JdbcTypeHandle type)
+    public static Optional<ColumnMapping> jdbcTypeToPrestoType(ConnectorSession session, JdbcTypeHandle type, Set<String> jdbcTypesMappedToVarchar)
     {
+        if (jdbcTypesMappedToVarchar.contains(type.getJdbcTypeName().get())) {
```

Comment:
don't fail when `type.getJdbcTypeName()` is empty

Reply:
it won't fail since `jdbcTypesMappedToVarchar` doesn't contain empty strings